Source/astcenc_vecmathlib.h

```diff
@@ -130,6 +130,7 @@ SIMD_INLINE vfloat loada(const float* p) { return vfloat(_mm256_load_ps(p)); }
 SIMD_INLINE vfloat operator+ (vfloat a, vfloat b) { a.m = _mm256_add_ps(a.m, b.m); return a; }
 SIMD_INLINE vfloat operator- (vfloat a, vfloat b) { a.m = _mm256_sub_ps(a.m, b.m); return a; }
 SIMD_INLINE vfloat operator* (vfloat a, vfloat b) { a.m = _mm256_mul_ps(a.m, b.m); return a; }
+SIMD_INLINE vfloat operator/ (vfloat a, vfloat b) { a.m = _mm256_div_ps(a.m, b.m); return a; }
```

Comment:
Not one for this PR, but if we're happy with a fast approximation it might be worth trying rcp and mul here, as div is quite a lot slower (13 cycles on Haswell for div, vs 2.5 cycles for rcp + mul).

Reply:
Alternatively, providing a fastdiv function so it can be used selectively.

Reply:
Maybe as an option, but precision of rcp is different between Intel & AMD CPUs, and I'd rather avoid that. We have tweaked another texture compression (ISPC one) to avoid this for exact this reason.

Reply:
Didn't know that one - that's a pain.